src/components/ClusterPage/index.jsx

```diff
@@ -198,14 +198,14 @@ ClusterPage.propTypes = {
   message: PropTypes.string.isRequired
 };
 
-const mapStateToProps = ({ user, AddClusterReducer }) => {
+const mapStateToProps = ({ user, addCluster }) => {
```

Comment:
@StephenAraka please explain this to me, from the code, it looks like `addCluster` is an action, and you are getting `reducer` variables from it? Please help me understand it

Reply:
@Mubangizi  there's addCluster the reducer and addCluster the action...
In that case, mapStateToProps refers to addCluster the reducer.

I renamed the reducer function.

Reply:
No please append `Reducer` to it, for uniformity. like all the other `reducerNames`

Reply:
done